src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -115,15 +115,18 @@
    * Creates a new FirebaseUserManager instance.
    *
    * @param app A non-null {@link FirebaseApp}.
+   * @param tenantId The associated tenant ID if the user operations should be tenant-aware,
+   *     otherwise {@code null}
    */
-  FirebaseUserManager(@NonNull FirebaseApp app) {
+  FirebaseUserManager(@NonNull FirebaseApp app, @Nullable String tenantId) {
```

Comment:
For readability, I'd suggest defining another constructor for this class:

```
FirebaseUserManager(@NonNull FirebaseApp app) {
  this(app, null);
}
```

You can call it from `FirebaseAuth`.

Reply:
Done.